Framework/Indexing/inc/MantidIndexing/IndexInfo.h

```diff
@@ -10,7 +10,11 @@
 #include <vector>
 
 namespace Mantid {
+
+using specnum_t = int32_t;
+using detid_t = int32_t;
```

Comment:
The indexing module does not use those typedefs, there are type-safe types provided for this purpose, `SpectrumNumber` and `DetectorID`. Not sure why this was added to the header, maybe an oversight?